tools/build_pytorch_libs.sh

```diff
@@ -287,7 +287,11 @@ function build_caffe2() {
       fi
 
       for proto_file in $(pwd)/caffe2/proto/*.py; do
-          cp $proto_file "$(pwd)/../caffe2/proto/"
+          # __init__.py is not auto-generated, copying it breaks
```

Comment:
I'm guessing we need to do this for Windows as well in build_pytorch_libs.bat? It looks like this doesn't exist yet, but since we've removed it from setup.py we'd need to add it.

Reply:
All windows tests passed, so there would be not way to get test signal on any change to the bat file.